pkg/groups/gsuite/grouper.go

```diff
@@ -109,7 +109,9 @@ func (g *Grouper) Members(group string) ([]rbacv1.Subject, error) {
 	ctx := context.TODO()
 	client, err := g.service(ctx)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(groups.ErrTemporary,
```

Comment:
This needs to differentiate between the errors returned on the service. For example, a not found should translate to a not found error. You might have to destructure the error and observe some fields defined by the upstream package.

Reply:
Wait, not here. I'll look down below.